go.sum

```diff
@@ -664,8 +664,9 @@ golang.org/x/text v0.3.1-0.20171227012246-e19ae1496984/go.mod h1:NqM8EUOU14njkJ3
 golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2/go.mod h1:NqM8EUOU14njkJ3fqMW+pc6Ldnwhi/IjpwHt7yyuwOQ=
 golang.org/x/text v0.3.2/go.mod h1:bEr9sfX3Q8Zfm5fL9x+3itogRgK3+ptLWKqgva+5dAk=
 golang.org/x/text v0.3.3/go.mod h1:5Zoc/QRtKVWzQhOtBMvqHzDpF6irO9z98xDceosuGiQ=
-golang.org/x/text v0.3.4 h1:0YWbFKbhXG/wIiuHDSKpS0Iy7FSA+u45VtBMfQcFTTc=
```

Comment:
The go.sum file lists all the direct and indirect dependencies of the program. Some dependencies are using different versions of the same dependency, that's why it's listing multiple versions of the golang.org/x/text.
Only one line was removed, the one without the `/go.sum`, because that is the line related to the hash of the zip file of the version of the dependency cached and used by the application.

For example, listing the dependency before the version bump:
```
% go version -m bin/gke-operator | grep text
	dep	golang.org/x/text	v0.3.4	h1:0YWbFKbhXG/wIiuHDSKpS0Iy7FSA+u45VtBMfQcFTTc=
```

After the version bump:
```
% go version -m bin/gke-operator | grep text
	dep	golang.org/x/text	v0.3.7	h1:olpwvP2KacW1ZWvsR7uQhoyTYvKAupfQrRGBFM352Gk=
```